runtime/JavaScript/src/antlr4/Lexer.js

```diff
@@ -121,7 +121,9 @@ export default class Lexer extends Recognizer {
 							this.notifyListeners(e); // report error
 							this.recover(e);
 						} else {
-							console.log(e.stack);
+                            if (this._interp.debug) {
```

Comment:
This code hasn't changed in 4 years, so the change in behavior relates to something else. Maybe some grammar specific code (predicate or production) that throws this exception, most probably unrelated to antlr itself ?

Reply:
I will investigate and look for independent reproduction.